README.md

```diff
@@ -44,7 +44,6 @@ docker compose up -d
 
 You need to have installed [Node](https://nodejs.org/en/about/previous-releases) Version 18.
 
-**WARNING** In linux with nodejs Version 18 the installation could be a challenge.
```

Comment:
Hi, Thank you for the PR, please leave the WARNING, in fact I have just extended for problems with odbc installation